rdr_service/offline/study_nph_ingest_awardee_data.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import sys
+import logging
+from os.path import isabs, exists
+from typing import Dict, Any, Iterator, Optional
+from io import StringIO
+from csv import DictReader
+
+from rdr_service.services.system_utils import setup_logging, setup_i18n
+from rdr_service.model.study_nph import Site
+from rdr_service.dao.study_nph_dao import NphSiteDao
+
+
+_logger = logging.getLogger("rdr_logger")
+
+# Tool_cmd and tool_desc name are required.
+# Remember to add/update bash completion in 'tool_lib/tools.bash'
+tool_cmd = "study_nph_ingest_awardee_data_from_file"
+tool_desc = "NPH Study Ingest Awardee Data"
+
+def read_csv(filepath: str) -> Iterator[Dict[str, Any]]:
+
+    def _decode_utf8_sig_content(csv_fp) -> StringIO:
+        content = csv_fp.read()
+        content = content.decode("utf-8-sig")
+        return StringIO(content)
+
+    with open(filepath, "rb") as csv_fp:
```

Comment:
Rather than having a `_decode_utf8_sig_content` method, would it work to give the `open` method the info for decoding?
```suggestion
    with open(filepath, "rb", encoding="utf-8-sig") as csv_fp:
```

Reply:
@kenny-skaggs, I pushed the suggested change. I updated it to `with open(filepath, "r", encoding="utf-8-sig") as csv_fp`